.ci/validation.sh

```diff
@@ -1128,6 +1128,22 @@ ci-temp-check)
     exit ${#FILES_NO_CONCURRENCY[@]}
     ;;
 
+check-wildcards-on-pitest-target-classes)
+  ALL_CLASSES=$(xmlstarlet sel \
+    -N x=http://maven.apache.org/POM/4.0.0 \
+    -t -v "/x:project/x:profiles/x:profile//x:targetClasses/x:param" \
```

Comment:
You don't need to specify profile id (starts with `pitest-`) or such to ensure we avoid conflicts?

Reply:
I would rather wait for false positive to appear and update this script than to make this expression more complicated than it has to be. What do you think?

Reply:
I think it makes sense to put in this one ID check.